go/client/cmd_account_lockdown.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"errors"
+	"fmt"
+	"text/tabwriter"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+type CmdAccountLockdown struct {
+	libkb.Contextified
+	SetLockdownMode *bool
+	History         bool
+	Force           bool
+}
+
+func NewCmdAccountLockdown(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	cmd := &CmdAccountLockdown{
+		Contextified: libkb.NewContextified(g),
+	}
+	flags := []cli.Flag{
+		cli.BoolFlag{
+			Name:  "enable",
+			Usage: "Enable account lockdown mode.",
+		},
+		cli.BoolFlag{
+			Name:  "disable",
+			Usage: "Disable account lockdown mode.",
+		},
+		cli.BoolFlag{
+			Name:  "history",
+			Usage: "Print history of lockdown mode changes.",
+		},
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Don't prompt.",
+		},
+	}
+	return cli.Command{
+		Name:  "lockdown",
+		Usage: "Manage lockdown mode",
+		Flags: flags,
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(cmd, "lockdown", c)
+		},
+		Description: `Lockdown mode can be enabled for any Keybase account with one of
+   provisioned devices. Enabling lockdown mode does not add a link to
+   user's sigchain. Lockdown mode is an extension of Keybase API
+   server access control.
```

Comment:
I think it's better to start with "When lockdown mode is enabled, some op..." since that's the main virtue of lockdown mode and provides some context. Move this first paragraph of details to the end.

In the first sentence "with one of provisioned devices" -> something else, or delete the sentence since you'd have to be on a device to be able to run this command anyway.

Reply:
thanks, I reworded that thing according to your suggestions. let me know if it can still be improved